classroom-template-bundle-spec.md

```diff
@@ -0,0 +1,168 @@
+# Qwiklabs ClassroomTemplate Bundle Specification
+
+**Version 1**
+
+> This is a DRAFT document. We welcome feedback as this format evolves.
+
+## `qwiklabs.yaml` Structure
+
+Here's a sample `qwiklabs.yaml` file with all nested details removed to make it easier to see the general file structure.
+
+```yml
+entity_type: ClassroomTemplate
+schema_version: 1
+
+# ClassroomTemplate Attributes
+default_locale: en
+
+# Versions should be considered decorators, and are not used as a source of truth for revision history.
+version: ver. 2
+
+# one of ["Self-paced", "Bootcamp/Workshop", "Instructor-led"]
+classroom_type: Instructor-led
+
+course_code: T-AHYXXX-I
+
+title:
+  locales:
+    en: Architecting with Google Cloud Platform.
+
+description:
+  locales:
+    en: Learn GCP basics in a classroom setting.
+
+objectives:
+  locales:
+    en: |
+      <p>This course will teach people</p>
+      <ul>
+        <li>What GCP is</li>
+        <li>How to use some of its core components</li>
+      </ul>
+
+audience:
+  locales:
+    en: |
+      <p>This course is intended for people who</p>
+      <ul><li>Are new to GCP</li></ul>
+
+prerequisites:
+  locales:
+    en: <p>A basic understanding of serverless architecture is recommended but not required.</p>
+
+# Outline is a JSON formatted string
+outline:
+  locales:
+    en: >
+      {\"subhead\":\"The course includes presentations and hands-on labs.\",\"modules\":[{\"name\":\"Module 1: Anthos Overview\",\"items\":[\"Describe challenges of hybrid cloud\",\"Discuss modern solutions\",\"Describe the Anthos Technology Stack\"]}]}
+
+on_demand_url:
+  locales:
+    en: "https://www.coursera.org/specializations/architecting-hybrid-cloud-infrastructure-anthos"
+
+tags: [sample, life-changing, gcp]
+product_tags: ['compute engine', 'cloud storage']
+role_tags: ['cloud architect', 'developers backend']
+domain_tags: ['infrastructure']
+level: 1
+
+# Allow classroom scheduling for this ClassroomTemplate
+published: true
```

Comment:
Can we nix `published` from the spec and just set it to true if it comes from Alexandria?